clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -154,7 +155,8 @@ public class ConsumerConfig extends AbstractConfig {
      * <code>fetch.min.bytes</code>
      */
     public static final String FETCH_MIN_BYTES_CONFIG = "fetch.min.bytes";
-    private static final String FETCH_MIN_BYTES_DOC = "The minimum amount of data the server should return for a fetch request. If insufficient data is available the request will wait for that much data to accumulate before answering the request. The default setting of 1 byte means that fetch requests are answered as soon as a single byte of data is available or the fetch request times out waiting for data to arrive. Setting this to something greater than 1 will cause the server to wait for larger amounts of data to accumulate which can improve server throughput a bit at the cost of some additional latency.";
+    public static final int DEFAULT_FETCH_MIN_BYTES = 1;
+    private static final String FETCH_MIN_BYTES_DOC = "The minimum amount of data the server should return for a fetch request. If insufficient data is available the request will wait for that much data to accumulate before answering the request. The default setting of " + DEFAULT_FETCH_MIN_BYTES + " byte means that fetch requests are answered as soon as that many byte(s) of data is available or the fetch request times out waiting for data to arrive. Setting this to a larger value will cause the server to wait for larger amounts of data to accumulate which can improve server throughput a bit at the cost of some additional latency.";
```

Comment:
Assuming there's no content change here, but just github not smart enough with diffs :) LMK otherwise.

Reply:
I did change the doc string, actually. I updated it to use the value from `DEFAULT_FETCH_MIN_BYTES` and some minor rewording. You just have to scroll way over :)

```
< The default setting of 1 byte means that fetch requests are
> The default setting of " + DEFAULT_FETCH_MIN_BYTES + " byte means that fetch requests are
< answered as soon as a single byte of data is available
> answered as soon as that many byte(s) of data is available
```

I dunno. It may not be worth the hassle to change the wording.

Reply:
Thanks for letting me know! That's fine :)